src/switchroot/ostree-system-generator.c

```diff
@@ -49,6 +49,13 @@ main(int argc, char *argv[])
       exit (EXIT_SUCCESS);
   }
 
+  /* We conflict with the magic ostree-mount-deployment-var file for ostree-prepare-root */
+  { struct stat stbuf;
+    if (fstatat (AT_FDCWD, "/run/ostree-mount-deployment-var", &stbuf, 0) == 0)
```

Comment:
Let's maybe be nice and `unlinkat()` here in the interest of having `/run` be less littered.  Or in addition/alternatively, make the file `/run/ostree/initramfs-mount-var` since we already made `/run/ostree/` for the deployment staging bits.